AlphaWallet/TokenScriptClient/Models/TokenIdOrigin.swift

```diff
@@ -31,10 +31,14 @@ struct TokenIdOrigin {
             return String(numberEncodingUtf8String: number).flatMap { .string($0) }
         case .e18:
             return EtherNumberFormatter().number(from: String(number)).flatMap { .uint(BigUInt($0)) }
+        case .e8:
+            return EtherNumberFormatter().number(from: String(number)).flatMap { .uint(BigUInt($0)) }
         case .bool:
             return .bool(number != 0)
         case .void:
             return nil
+        default:
+            return nil
```

Comment:
Should we support functions returning `.bytes`, etc?

Reply:
Yes, I can add that in now